drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/main/java/org/drools/workbench/screens/scenariosimulation/webapp/client/dropdown/ScenarioSimulationKogitoRuntimeCreationAssetsDropdownProviderImpl.java

```diff
@@ -52,12 +52,20 @@ public void getItems(Consumer<List<KieAssetsDropdownItem>> assetListConsumer) {
         return response -> {
             List<KieAssetsDropdownItem> toAccept = response.stream()
                     .map(this::getKieAssetsDropdownItem)
+                    .filter(this::filterTargetDirectory)
                     .sorted(Comparator.comparing(KieAssetsDropdownItem::getText, String.CASE_INSENSITIVE_ORDER))
                     .collect(Collectors.toList());
             assetListConsumer.accept(toAccept);
         };
     }
 
+    protected boolean filterTargetDirectory(KieAssetsDropdownItem kieAssetsDropdownItem) {
+        String fullPath = kieAssetsDropdownItem.getValue().replaceAll("\\\\", "/");
+        int idx = fullPath.lastIndexOf('/');
+        final String pathString = idx >= 0 ? fullPath.substring(0, idx + 1) : "";
```

Comment:
Is it possible to create a `extractPath` method with this logic and use it also [here](https://github.com/kiegroup/drools-wb/blob/ecbde3e83c948a41fc983fad4cfb8238a3af3ae7/drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/main/java/org/drools/workbench/screens/scenariosimulation/webapp/client/dropdown/ScenarioSimulationKogitoRuntimeCreationAssetsDropdownProviderImpl.java#L77-L78)? It looks almost the same